docs/framework/winforms/controls/how-to-author-controls-for-windows-forms.md

```diff
@@ -43,7 +43,7 @@ A control represents a graphical link between the user and the program. A contro
   
 9. Repeat, refining the design.  
   
-10. Package and deploy your control. For details, see [Deploying Applications, Services, and Components](https://msdn.microsoft.com/library/wtzawcsz).  
+10. Package and deploy your control. For details, see [Deploying Applications, Services, and Components](/visualstudio/deployment/deploying-applications-services-and-components?view=vs-2015).  
```

Comment:
I'd avoid adding views and I'd stick to the default one since that will be the latest

```suggestion
10. Package and deploy your control. For details, see [First look at deployment in Visual Studio](/visualstudio/deployment/deploying-applications-services-and-components).  
```